joomla/joomla-cms
# Cassiopeia template - changing Tagline to Description - Optional text to show as a subheading parameter

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/35719

### Summary of Changes

Change the name of the parameter and add its description, as it was usual in Joomla 3. "Tagline" is replaced by "Description" and the description "Optional text to show as a subheading" is added.

![img](https://i.imgur.com/uGUWo2o.png)

### Testing Instructions

Go to Cassiopeia template options and see the parameter "Tagline" (before applying this PR) and "Description - Optional text to show as a subheading" (after applying this PR)

### Actual result BEFORE applying this Pull Request

"Tagline" displayed


### Expected result AFTER applying this Pull Request

"Description - Optional text to show as a subheading"

### Documentation Changes Required
